build_tools/benchmarks/run_benchmarks_on_linux.py

```diff
@@ -12,19 +12,24 @@
 # Add build_tools python dir to the search path.
 sys.path.insert(0, str(pathlib.Path(__file__).parent.with_name("python")))
 
-import subprocess
+from typing import Any, List, Optional
 import atexit
+import json
 import shutil
+import subprocess
 import tarfile
-
-from typing import List, Optional
+import typing
```

Comment:
I think Google style would put the `typing` imports together: https://google.github.io/styleguide/pyguide.html#313-imports-formatting. I thought it called for direct symbol imports in their own section, but it looks like I made that up.

Reply:
Done.